.circleci/common.yml

```diff
@@ -781,7 +778,7 @@ jobs:
             cp -r node_modules/ dist/node_modules/
 
             cd dist
-            zip -r9 ~/project/${CIRCLE_SHA1}.zip .
+            zip --symlinks -r9 ~/project/${CIRCLE_SHA1}.zip .
```

Comment:
We were missing this piece of the zip puzzle.

Reply:
aha